src/Workspaces/CSharp/Portable/Formatting/Engine/Trivia/TriviaDataFactory.Analyzer.cs

```diff
@@ -59,7 +59,7 @@ public static AnalysisResult Between(SyntaxToken token1, SyntaxToken token2)
                     // newline.  If so, we keep track of that so we'll appropriately indent later
                     // on. 
 
-                    var previousNonMissingToken = token1.GetPreviousToken(includeZeroWidth: false, includeSkipped: true);
```

Comment:
by the way, another way to solve this will be we don't convert tab<-> space if there is any syntax error on the span it is formatting.